src/nupic/support/decorators.py

```diff
@@ -49,16 +47,14 @@ def myFunctionFoo():
         raise RuntimeError("something bad happened")
         ...
   """
+  logger = (logger if logger is not None else logging.getLogger())
```

Comment:
Consistency: you used `logging.getLogger(__name__)` in the other modification in this module